src/initial.js

```diff
@@ -0,0 +1,12 @@
+const initial = input => {
+  const result = []
+
+  if ( !(input instanceof Array) ) {
+    return []
+  }
+    return input.splice(0, input.length-1)
```

Comment:
I think you want `slice`, otherwise you're changing the value of the passed array.  Can you do this without slice or splice?